manifests/subscription.pp

```diff
@@ -13,7 +13,9 @@
 
   include ::sensu
 
-  file { "${::sensu::conf_dir}/subscription_${name}.json":
+  $sanitized_name=regsubst($name, '[^0-9A-Za-z.-]', '_', 'G')
```

Comment:
Can you please comment on what this function is doing?

Reply:
regex explained in words can be really useful so you dont have to dissect what it is doing